rollup.config.js

```diff
@@ -6,19 +6,19 @@ export default {
   format: 'umd',
   plugins: [
     babel({
-        babelrc: false,
-        presets: [
-          "es2015-rollup"
-        ],
-        plugins: [
-          [
-            "transform-es2015-classes",
-            {
-              "loose": true
-            }
-          ]
+      babelrc: false,
+      presets: [
+        'es2015-rollup'
+      ],
+      plugins: [
+        [
+          'transform-es2015-classes',
+          {
+            loose: true
+          }
         ]
-      }
+      ]
+    }
```

Comment:
I see. In this case can you remove the line break before the closing `babel()` parens? Otherwise the closing bracket and parens are not indented and it looks weird.